daffodil-test-ibm1/src/test/resources/test-suite/ibm-contributed/dpaext1.tdml

```diff
@@ -730,7 +745,11 @@ entity NL and also use of UTF-16 encoding - DFDL-6-045R"
 
 <tdml:parserTestCase name="length_delimited_12_04"
  model="dpaextdeltxt101.dfdl.xsd" description="Section 12.3 Properties for specifying length -delimited - end of input"
- root="myDate2">
+ root="myDate2"
+ implementations="daffodil">
+  <!-- 
+   test doesn't run on IBM DFDL because of restriction that separtor/terminator must be unique CTDV1425W
```

Comment:
This restriction now holds for Daffodil also. Fix comment. Maybe the test should not be daffodil-specific?
 Same comment applies to other tests in this file.

Reply:
Fixed for all tests in the file.